perfkitbenchmarker/benchmarks/object_storage_service_benchmark.py

```diff
@@ -529,8 +537,10 @@ def Prepare(self, vm):
     vm.PushFile(FLAGS.boto_file_location, DEFAULT_BOTO_LOCATION)
 
     self.bucket_name = 'pkb%s' % FLAGS.run_uri
+
     vm.RemoteCommand(
-        'aws s3 mb s3://%s --region=us-east-1' % self.bucket_name)
+        'aws s3 mb s3://%s --region=%s' % (
```

Comment:
I've done this for AWS. I looked for a while on the Azure site, but I couldn't find any indication that we need to use different endpoints for different regions. I think they do DNS magic to always route you to a nearby endpoint from the same URL (and hopefully your object is in a bucket near you...).